src/sass/components/_tour.scss

```diff
@@ -29,3 +29,7 @@
   }
 }
 /* stylelint-enable declaration-no-important */
+
+.slds-popover_walkthrough .slds-popover__header {
+  background-image: url('~img/popover-header.png');
+}
```

Comment:
Background image `url(...)` calls from within the SLDS CSS file don't work, so we override them here.